applications/nrf5340_audio/src/utils/error_handler.c

```diff
@@ -14,23 +14,37 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(error_handler, CONFIG_ERROR_HANDLER_LOG_LEVEL);
 
-#if (defined(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP) && (CONFIG_DEBUG))
+#if defined(CONFIG_DEBUG)
```

Comment:
If debug is not enabled we have no LEDs?

Reply:
True and that is how it is currently, so I have not changed this.
I did wonder why this was the case!
Shall I remove this so as to always have the LEDs, if available?